modules/tinymce/src/models/dom/demo/ts/demo/Demos.ts

```diff
@@ -0,0 +1,7 @@
+import BasicDemo from './BasicDemo';
+
+declare let window: any;
+
+window.demos = {
+  BasicDemo
+};
```

Comment:
Do we actually need something as "complex" as this for the `dom` model demo? This pattern is normally only used where we're going to need multiple demos (e.g. core uses this but plugins do not) and I don't think that applies here so we could just have the single `Demo.ts` file like in plugins.